### PR TITLE
fix(headless-cms): content models via code immutable definitions

### PR DESCRIPTION
**Pull request by brunozoric**
## Short Description
Warning about changing model via code definitions.

## Relevant Links

- [Immutable Code Definitions in Model](https://docs-webiny-ck0iavwaw-webiny.vercel.app/docs/headless-cms/extending/content-models-via-code#immutable-code-definitions)

## Checklist
- [ ] I added page metadata (description, keywords)
- [ ] I've added "Can I Use This?" section (if needed, e.g. if documenting a new feature)
- [ ] I added `What You'll Learn` at the top of the page and every item in the list starts with a lower case letter
- [ ] I used title case for titles and subtitles (in the main menu and in the page content)
- [ ] I checked for typos and grammar mistakes
- [ ] I added `alt` / `title` attributes for inserted images (if any)
- [ ] When linking code from GitHub, I did it via a specific tag (and not `next` / `v5` branch) 

## Screenshots (if relevant):
N/A

### CODE REVIEW

File: src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx [hunk 1]
```diff
@@ -375,3 +375,25 @@ Here are a couple of examples of specifying icons from solid, regular, and brand
 - `far/copy`
 - `fab/aws`.
 - `fab/react`.
+
+## Immutable Code Definitions
+
+As you can do what ever you want with the models via code, there are things you need to be really careful about.
+We cannot prevent you to change anything, so you need to be watchful when creating models via code.
```
Comment (endymion1818) [inline src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx:382]:
```suggestion
You can do what ever you want with the models via code. For that reason, there are things you need to be really careful about. We cannot prevent you from changing anything, so you need to be watchful for these pitfalls.
```

File: src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx [hunk 2]
```diff
@@ -375,3 +375,25 @@ Here are a couple of examples of specifying icons from solid, regular, and brand
...
+There are also few field properties which must **never** change, if you want to keep your data.
+
+Properties are:
+ - `id`
+ - `storageId` - if you have created it manually
+ - `type`
+ - `multipleValues`
+ - `predefinedValues` - you can add values but do not remove them
+
+Also, you have defined anything settings, be careful about these properties:
```
Comment (endymion1818) [inline src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx:395]: Is there an `if` missing here?
```suggestion
Also, if you have defined anything settings, be careful about these properties:
```

File: src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx [hunk 3]
```diff
@@ -375,3 +375,25 @@ Here are a couple of examples of specifying icons from solid, regular, and brand
...
+
+ - `settings.fields` - same rule applies as for the parent field
```
Comment (endymion1818) [inline src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx:397]: Same what rule? What's the parent?

Comment (brunozoric) [reply src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx:396]: parent field being the one you have defined settings.fields in?

**Pull request merged**